### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Storage Container_Screen Reader: Narrator/NVDA is not differentiating between the folder file and a non-folder file present in the table

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3579]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS1.3.1; #ScreenReader; #C+E-H; #JavatoolingforIntelliJ/Eclipse; #Narrator; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator, NVDA
MAS Violated: MAS1.3.1

**Repro Steps:** 
1) Start Narrator/NVDA. Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) Tab till Windows menu option-->Show View-->Other-->Expand Azure-->Azure Explorer pane press OK
6) In Azure Explorer section, tab till HDInsight and expand it. Expand the cluster name
7) Tab till the storage container and press enter
8) In Storage Container Section, when focus lands on the table item, verify if Narrator/NVDA is differentiating between the folder file and a non-folder file present in the table

**Actual Result:** 
Narrator/NVDA is not differentiating between the folder file and a non-folder file present in the table

**Expected Result:** 
Narrator/NVDA is must differentiate between the folder file and a non-folder file present in the table so user can select a folder related file

**User Impact:** 
Screen reader users will find it difficulty in understanding which folder to select if screen reader does not read the difference between folder file and non-folder file

**MAS Reference:** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_9_New.pptx](https://github.com/microsoft/azure-tools-for-java/files/3640754/Bug_9_New.pptx)

**Comment by t-rufang** [comment]
Note: Screen reader is not differentiating between the folder file and a non-folder file present in the table
![image](https://user-images.githubusercontent.com/32627233/65828573-d5d29980-e2ce-11e9-94ef-ca925285ef8b.png)

**Comment by Flanker32** [comment]
@Tanushagottipati We have removed storage explorer in IDE toolkit, could you please help us verify and close the issue? Here is the [build](https://dev.azure.com/mseng/VSJava/_build/results?buildId=11079936)

**Comment by Tanushagottipati** [comment]
As mentioned storage accounts tab cannot be opened in the eclipse toolkit. As storage explorer has been deprecated, closing this bug and attached screenshot for reference.
Environment:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-09 R (4.13.0)
Build id: 20190917-1200
Solution Installed: The Azure Toolkit for Eclipse 3.20.0
![Screenshot (709)](https://user-images.githubusercontent.com/41109135/70886693-1904f480-2002-11ea-82d4-8db90beb85e2.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749683
